src/Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -69,6 +69,32 @@ static PowerShellKernel()
             // Add accelerators that exist in other namespaces.
             addAccelerator.Invoke(null, new object[] { "Layout", typeof(Layout.Layout) });
             addAccelerator.Invoke(null, new object[] { "Chart", typeof(Chart) });
+
+            //TODO: Is this declared anywhere else?
+            string[] mimeTypes = new string[] { HtmlFormatter.MimeType, JsonFormatter.MimeType, PlainTextFormatter.MimeType };
+
+            foreach (string mt in mimeTypes)
```

Comment:
It seems the rebase broke the changes. The formater code should be removed already.

Reply:
I forgot to fetch the upstream master; will be redoing the rebase